tests/components/light/test_init.py

```diff
@@ -18,479 +18,479 @@
     STATE_ON,
 )
 from homeassistant.exceptions import Unauthorized
-from homeassistant.setup import async_setup_component, setup_component
+from homeassistant.setup import async_setup_component
 
 import tests.async_mock as mock
-from tests.common import get_test_home_assistant, mock_service, mock_storage
+from tests.common import async_mock_service
 from tests.components.light import common
 
 
-class TestLight(unittest.TestCase):
-    """Test the light module."""
+@pytest.fixture
+def mock_storage(hass, hass_storage):
+    """Clean up user light files at the end."""
+    yield
+    user_light_file = hass.config.path(light.LIGHT_PROFILES_FILE)
 
-    # pylint: disable=invalid-name
-    def setUp(self):
-        """Set up things to be run when tests are started."""
-        self.hass = get_test_home_assistant()
-        self.addCleanup(self.tear_down_cleanup)
+    if os.path.isfile(user_light_file):
+        os.remove(user_light_file)
 
-    def tear_down_cleanup(self):
-        """Stop everything that was started."""
-        self.hass.stop()
 
-        user_light_file = self.hass.config.path(light.LIGHT_PROFILES_FILE)
+async def test_methods(hass):
+    """Test if methods call the services as expected."""
+    # Test is_on
+    hass.states.async_set("light.test", STATE_ON)
+    assert light.is_on(hass, "light.test")
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6239266)